src/main/resources/project.properties

```diff
@@ -0,0 +1,7 @@
+project.version=1.0-SNAPSHOT
```

Comment:
No native way to get version from pom.xml since the java project is not aware of the file. There's a plugin for updating the version by reading the pom.xml, but I think that's a bit overkill.